oap-server/server-starter/src/main/resources/ui-initialized-templates/elasticsearch/elasticsearch-index.json

```
@@ -0,0 +1,796 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+[
+  {
+    "id": "Elasticsearch-Index",
+    "configuration": {
+      "children":[
+        {
+          "x":0,
+          "y":6,
+          "w":12,
+          "h":13,
+          "i":"0",
+          "type":"Widget",
+          "id":"0",
+          "metricTypes":[
+            "readMetricsValues",
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_stats_indexing_index_total_req_rate",
+            "meter_elasticsearch_index_stats_indexing_index_total_proc_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":true,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Indexing Rate",
+            "tips":"`proc rate`: processing ability, which is the number of requests that Elasticsearch can handle. Estimation is based on the processed requests and processed time for these requests."
+          },
+          "metricConfig":[
+            {
+              "label":"req rate"
```

```suggestion
              "label":"req_rate"
```
Can we connect the words with `_`, then could try to keep all styles with others as consistent as possible